backend/open_webui/retrieval/loaders/youtube.py

```
@@ -101,17 +106,31 @@ def load(self) -> List[Document]:
+                continue
+            except Exception as e:
+                log.info(f"Error finding transcript for language '{lang}'")
+                raise e
+
+        # If we get here, all languages failed
+        languages_tried = ", ".join(self.language)
+        log.warning(
+            f"No transcript found for any of the specified languages: {languages_tried}. Verify if the video has transcripts, add more languages if needed."
+        )
+        raise NoTranscriptFound(
+            f"No transcript found for any supported language. Verify if the video has transcripts, add more languages if needed."
```

Ruff (0.12.2)

f-string without any placeholders

Remove extraneous `f` prefix

(F541)